test/unit/config/matchers/to-be-positioned-popover.js

```diff
@@ -5,10 +5,19 @@
  * @param {HTMLElement} element Popover element.
  */
 function toBePositionedPopover( element ) {
-	const pass = element.classList.contains( 'is-positioned' );
+	const popover = element && element.closest( '.components-popover' );
+	const isPopoverPositioned =
+		popover && popover.classList.contains( 'is-positioned' );
+	const pass = !! popover && !! isPopoverPositioned;
+
 	return {
 		pass,
-		message: () => `Received element is ${ pass ? '' : 'not ' }positioned`,
+		message: () => {
+			const is = pass ? 'is' : 'is not';
+			return ! popover
+				? `Received element ${ is }, or ${ is } positioned inside, a Popover component.`
```

Comment:
A better wording IMO would be "Received element { is not } a popover element or its descendant."